### DISCUSSION TITLE
Lesson 2: SimpleStorage

### DISCUSSION THREAD

**Discussion by rotradam** [smartcontractkit/full-blockchain-solidity-course-py#72] [Q&A] [answered]
Somehow I get this following error when I deploy.

![image](https://user-images.githubusercontent.com/91180828/134297464-9e8bcb04-c165-48c2-bec1-9b639e363c4d.png

Can anyone help me figure out the error? 
Thanks!! :)

**Comment by rotradam** [comment, marked as answer]
Ahh, I just figured out the error. I saved the function as 'Store' on the first file (Storage.sol) and 'store' (here on the Factory.sol). I forgot how case-sensitive everything is.

I'm starting to code after 6-8 years and I only have experience with Matlab and c++. I'm jumping straight to Solidity, I know it will take some time for me to catch up, but slowly I'm getting there.

**Reply by PatrickAlphaC** [reply]
Awesome! Welcome to the space! You’re already doing the right thing jumping in here.
